docs/en/observability/user-experience.asciidoc

```diff
@@ -0,0 +1,194 @@
+// At some point we need to add a page that talks about how synthetics and
+// RUM/UE work together. Passive/Active, etc.
+
+[[user-experience]]
+= {user-experience}
+
+{user-experience} provides a way to quantify and analyze the perceived performance of your web application.
+Unlike testing environments, {user-experience} data reflects actual users' real-world experiences.
+Powered by the APM Real user monitoring (RUM) agent, all it takes is a few lines of code to begin
+surfacing key user experience metrics and investigative tools.
+Drill down further by looking at data by URL, device type, operating system, browser, and location --
```

Comment:
We don't support device type as a filter or break down yet. It's device make and model right now.